src/config.ts

```diff
@@ -44,11 +44,15 @@ interface Config {
   'VertexAI Api Domain Part'?: string;
   'Gemini Model'?: string;
   'Image Generation Model'?: string;
+  'Is Promotion Mode': string;
 }
 
 export const sheet =
   SpreadsheetApp.getActiveSpreadsheet().getSheetByName('Config')!;
-const DEFAULT_CONFIG = {
+export const promotionSheet =
+  SpreadsheetApp.getActiveSpreadsheet().getSheetByName('Promotion_Config')!;
```

Comment:
I think you can make the setup even more flexible if you use in the main config sheet `Promotion Mode Config` = <PROMO_CONFIG_SHEET_NAME> (if empty it is equal to `Is Promotion Mode` = no) instead of `Is Promotion Mode` = yes|no. In this way you can have several configs for e.g. different promo seasons and quickly switch them.

Reply:
I totally agree! fixed in the new commit